chapter2/tests/t14/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10.11-slim-buster AS builder
+
+WORKDIR /app
+COPY memory.py requirements.txt ./
+RUN apt-get update && apt-get install -y gcc=4:8.3.0-1 python3-dev=3.7.3-1 musl-dev=1.1.21-2 --no-install-recommends
+RUN pip install --no-cache-dir -r requirements.txt --upgrade pip && pip install --no-cache-dir -r requirements.txt
```

Comment:
> format properly - use multiline. Oneliners are hard to read.

@dmitry-mightydevops 
I just had to do it because the pre-commit gave a warning about combining commands, and I shouldn't have made a PR if the files didn't pass the check, although I knew it looked worse..

Reply:
@rtest12 it has ignore rules - use it
https://github.com/hadolint/hadolint#cli

Reply:
> format properly - use multiline. Oneliners are hard to read.

@dmitry-mightydevops 
Done